src/parse.c

```diff
@@ -725,12 +739,59 @@ static ident_t loc_to_ident(const loc_t *loc)
    return ident_new(tb_get(tb));
 }
 
+static ident_t get_implicit_label(tree_t t, const loc_t *loc)
+{
+   int *cnt;
+   char c;
+
+   switch (tree_kind(t)) {
+   case T_PROCESS:
+   case T_CONCURRENT:
+      cnt = &(imp_label_cnts.proc);
+      c = 'P';
+      break;
+
+   case T_FOR:
+   case T_WHILE:
+      cnt = &(imp_label_cnts.loop);
+      c = 'L';
+      break;
+
+   case T_WAIT:
```

Comment:
Make this the `default:` and get rid of `loc_to_ident` as I don't think it's used anywhere else.

Reply:
Done, code moved to `get_implicit_label` in `names.c`. There was only remaining, use of it, now commented
with TODO, is there a need to keep name of sequence of statements under case alternative?